modules/custom/drupalmel_core/drupalmel_core.install

```diff
@@ -81,6 +81,7 @@ function drupalmel_core_update_7000() {
   $feeds = array(
     'Kattekrab'   => 'http://kattekrab.net/taxonomy/term/322/feed',
     'Realityloop' => 'http://realityloop.com/tags/drupal-planet/feed',
+    'Flink' => 'http://flink.com.au/taxonomy/term/1/feed',
```

Comment:
So something like:

    function drupalmel_core_update_7001() {
      // Enable persistent update module.
      module_enable(array('persistent_update'));
    
      // Run this modules persistent update hook.
      drupalmel_core_persistent_update();
    
      // Add blog feeds.
      $feeds = array(
        'Flink' => 'http://flink.com.au/taxonomy/term/1/feed',
      );
    }
?